_i18n/ar/resources/moneropedia/p2pool.md

```diff
@@ -0,0 +1,42 @@
+---
+entry: "P2Pool"
+terms: ["P2Pool", "monero-p2pool"]
+summary: "Peer to peer mining pool for Monero"
+---
+
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+### The Basics
+
+Monero P2Pool is a peer-to-peer Monero @mining pool developed by SChernykh (also known as sech1).
```

Comment:
If there aren't two spaces after a string, markdown won't add a newline. I would suggest to avoid the double space or translators will have to deal with multiple `\n` in the strings. Better either add a newline or everything in the same string.

Reply:
I think you're referring to the /n in the middle of the paragraph here. Per @hyc on my last PR, long lines are harder for reviewers to give feedback on. If keeping the paragraph on one line is preferred I can do it that way. In this case line 11 and line 12 are the same paragraph, and I'd lean towards making the translators lives easier.

Reply:
I'm re-reading this 8 hours later I'm still not sure I'm tracking correctly. I'm assuming that you mean don't do multiple lines inside of paragraphs. I intended that the single /n would render all in same paragraph and did /n/n for paragraph breaks, but I'm understanding that makes translation harder. Let me know after I make my next commit if I've misunderstood.

Reply:
Now looks good :)